### PR TITLE
[ticket/14107] Fix dropdown rtl states

### PR DESCRIPTION
**Pull request by hanakin**
[ticket/14107] Fix dropdown rtl states

https://tracker.phpbb.com/browse/PHPBB3-14107
PHPBB3-14107

### CODE REVIEW

**Comment by hanakin** [comment]
!set Styles

File: phpBB/styles/prosilver/theme/bidi.css [hunk 1]
```diff
@@ -114,13 +114,9 @@
...
 
-.rtl .dropdown-extended .header .header_settings {
+.rtl .dropdown-extended .header .header_settings, .rtl .dropdown-container-right {
 	float: left;
-}
-
-/* Notifications
-----------------------------------------*/
-.rtl .notification_list ul li img {
+}rtl .notification_list ul li img {
```
Comment (Nicofuma) [inline phpBB/styles/prosilver/theme/bidi.css]: You removed the comment?

File: phpBB/styles/prosilver/theme/bidi.css [hunk 2]
```diff
@@ -114,13 +114,13 @@
...
-.rtl .dropdown-extended .header .header_settings {
+.rtl .dropdown-extended .header .header_settings, .rtl .dropdown-container-right {
 	float: left;
 }
 
 /* Notifications
-----------------------------------------*/
-.rtl .notification_list ul li img {
+-----------------------------------------*/
+rtl .notification_list ul li img {
```
Comment (marc1706) [inline phpBB/styles/prosilver/theme/bidi.css]: Removing the dot in front of rtl doesn't seem to make any sense to me? Typo?

**Comment by hanakin** [comment]
This is ready for review @cyberalien

**Comment by cyberalien** [comment]
In topic at bottom of page tools dropdown is on left side with RTL language. It should be on right side.

**Comment by cyberalien** [comment]
Jumpbox is also on wrong side

**Comment by hanakin** [comment]
this is what topic looks like for me with rtl
screenshot 2015-08-27 05 56 12

is this not what you are seeing?

**Comment by cyberalien** [comment]
No, this is what I'm seeing:

rtl

Its a fresh installation directly from your branch. 

In LTR language jumpbox is wrapped in dropdown-container-right that has float:right.
In RTL language jumpbox is wrapped in dropdown-container-left that normally has float:left, but in bidi.css it is overwritten with float:right

So in both directions it floats to right. Same with quickmod dropdown.

**Comment by hanakin** [comment]
have to look into this more when i get home tonight as that makes no sense.

dropdown-container-right = float:right
dropdown-container-left = float: left

in bidi they are switched so that they float to the opposite side I think you may have cache issues

**Comment by cyberalien** [comment]
No, there are no cache issues. It was fresh installation directly from your repo in new directory, there is nothing to cache.

**Comment by cyberalien** [comment]
From jumpbox.html:

    <div class="dropdown-container dropdown-container-{S_CONTENT_FLOW_END}

From bidi.css:

    .rtl .dropdown-container.topic-tools, .rtl .dropdown-container-left {
    	float: right;
    }


First it changes direction in HTML code by changing class name, so float:right becomes float:left. Then it again changes direction in bidi.css, which reverts it back to float:right.

**Comment by hanakin** [comment]
ah thats why I was not aware the it was changing it in the html in this way. this seems counter intuitive

**Comment by hanakin** [comment]
@cyberalien can please have another look at this

**Comment by marc1706** [comment]
Looks good now.

**Comment by cyberalien** [comment]
.jumpbox .dropdown-contents a

has margin on wrong side. fix:

    .rtl .jumpbox .dropdown-contents a { margin-right: 0; margin-left: 20px; }

everything else looks good

**Pull request merged**